services/wallet/collectibles/types.go

```diff
@@ -32,9 +32,15 @@ type CollectibleData struct {
 }
 
 type CollectionData struct {
-	Name     string `json:"name"`
-	Slug     string `json:"slug"`
-	ImageURL string `json:"image_url"`
+	Name     string            `json:"name"`
+	Slug     string            `json:"slug"`
+	ImageURL string            `json:"image_url"`
+	Socials  CollectionSocials `json:"socials"`
```

Comment:
`Socials` might not always be available, so this should be a pointer.
That way, nil Socials means "no Socials fetched yet", as opposed to a valid Socials with empty string fields.